docs/crocks/Maybe.md

```diff
@@ -0,0 +1,97 @@
+# Maybe
+
+`Maybe c a`
+
+--
+
+--
+
+```js
+--
+```
+
+`Maybe` exposes the following constructor and instances:
+
+| Constructor | Instance |
+|:---|:---|
+| [`Nothing`](#nothing), [`Just`](#just), [`of`](#of) | [`inspect`](#inspect), [`maybe`](#maybe), [`either`](#either), [`option`](#option), [`type`](#type), [`equals`](#equals), [`coalesce`](#coalesce), [`map`](#map), [`ap`](#ap), [`of`](#of), [`chain`](#chain), [`sequence`](#sequence), [`traverse`](#traverse) |
+
+## Constructors
+
+### Nothing
+
+`Maybe m => c -> m c _`
```

Comment:
The first parameter is fixed to `()`, so this should be:

`Maybe m => _ -> m a`